packages/datadog-plugin-aws-sdk/src/services/cloudwatchlogs.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const BaseService = require('./base')
+
+class CloudwatchLogsService extends BaseService {
+  _addServiceTags (params, operation, response) {
+    const tags = {}
+
+    // cloudwatach log group name
```

Comment:
Comment is redundant since the variable is named exactly the same as the comment.

Reply:
There are several other occurrences in other services as well.

Reply:
yup, removed basically all the comments in these services as either there's now an appropriate test for the behavior or the naming itself makes it redundant